src/config/locales/en-US.json

```diff
@@ -783,7 +783,9 @@
     "line2_body":"Utilizing blockchain, censorship-free, decentralized and rewarding.",
     "line3_heading":"Join Ecency communities!",
     "line3_body":"Build community you own, get rewarded and reward others.",
-    "get_started":"Get started!"
+    "get_started":"Get started!",
+    "terms_description": "By accepting, you agree to our Terms of Service and Privacy Policies.",
+    "terms_text": " Read Here!"
```

Comment:
This space should be added into code if possible because translators will most likely forget to add that space when translating.